src/components/BaseCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="flex flex-col rounded-lg shadow-lg overflow-hidden">
+    <div class="flex-shrink-0">
+      <img class="h-48 w-full object-cover" :src="src">
+    </div>
+    <div class="flex-1 bg-white p-6 flex flex-col justify-between">
+      <div class="flex-1">
+        <p class="text-sm font-medium text-indigo-600">
+          <a
+              v-for="tag in tags"
+              :key="tag"
+              class="hover:underline mr-2">{{ tag }}</a>
+        </p>
+        <a :href="link" class="block mt-2">
+          <p class="text-xl font-semibold text-gray-900">
+            {{ title }}
+          </p>
+          <p class="mt-3 text-base text-gray-500">
+            {{ content }}
+          </p>
+        </a>
+      </div>
+      <div class="mt-6 flex items-center">
+        <div class="flex-shrink-0">
+          <a href="#">
+            <span class="sr-only">{{ author }}</span>
+            <img class="h-10 w-10 rounded-full" :src="avatar">
+          </a>
+        </div>
+        <div class="ml-3">
+          <p class="text-sm font-medium text-gray-900">
+            <a href="#" class="hover:underline">
+              {{ author }}
+            </a>
+          </p>
+          <div class="flex space-x-1 text-sm text-gray-500">
+              <span>
+                  {{ views }} {{ $t('views') }}
+                </span>
+            <span aria-hidden="true">
+                  &middot;
+                </span>
+            <span>
+                  x {{ $t('minutes') }} {{ $t('read') }}
+                </span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BaseCard',
+  props: {
```

Comment:
Rather than having a prop for every field just send the card as an object and using . notation and you can simplify the code